### PR TITLE
Fix return type of initiate_step

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: src/isar_exr/robotinterface.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __init__(self):
             os.path.dirname(os.path.realpath(__file__)), "example_images"
         )
 
-    def initiate_step(self, step: Step) -> bool:
+    def initiate_step(self, step: Step) -> None:
         time.sleep(STEP_DURATION_IN_SECONDS)
         return True
```
Comment (GodVenn) [inline src/isar_exr/robotinterface.py:63]:
```suggestion
        return None
```

**Review by GodVenn — APPROVED** [review]
LGTM

**Review by eivindsjovold — APPROVED** [review]
LGTM

**Comment by UsamaEquinorAFK** [comment]
Closes #26

**Pull request closed without merging**